cmd/snap/cmd_debug_timings_test.go

```diff
@@ -0,0 +1,204 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main_test
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/cmd/snap"
+)
+
+type timingsCmdArgs struct {
+	args, stdout, stderr, error string
+}
+
+var timingsTests = []timingsCmdArgs{{
+	args:  "debug timings",
+	error: "please provide change ID or type with --last=<type>",
+}, {
+	args:  "debug timings --ensure=foo 9",
+	error: "cannot use 'ensure' and change id together",
+}, {
+	args:  "debug timings --last=install --all",
+	error: "cannot use 'all' with change id or 'last'",
+}, {
+	args:  "debug timings --last=remove",
+	error: `no changes of type "remove" found`,
+}, {
+	args:  "debug timings --all 9",
+	error: "cannot use 'all' with change id or 'last'",
+}, {
+	args: "debug timings --last=install",
+	stdout: "ID   Status        Doing      Undoing  Summary\n" +
+		"40   Doing         910ms            -  .\n" +
```

Comment:
What will there be usually shown? Instead of the `.` ? If possible it would be nice to mock to something slightly more real here. Maybe just "bar summary"?